backend/tests/icpy/agent/tools/test_atlascloud_ttv_tool.py

```
@@ -0,0 +1,467 @@
+        assert client1 is client2  # Same instance
+
+
+# Integration tests (requires real API key)
+@pytest.mark.integration
+@pytest.mark.asyncio
+async def test_real_api_generation():
+    """
+    Integration test with real API (requires ATLASCLOUD_API_KEY).
+    Marked as integration test - skip by default.
+    """
+    import os
```

Ruff (0.14.14)

[error] Redefinition of unused `os` from line 12: `os` redefined here

Remove definition: `os`

(F811)